### ISSUE TITLE
Not allowed to see photo's with non-public permissions [master]

### ISSUE THREAD

**Issue by jeroenpraat** [redmatrix/hubzilla#483]
[This occurs only in master and not in 1.12RC]

When permissions of photo's are set to anything other than public (like 'only me' or your default privacy group) you are not allowed to see the photo (the famous 'forbidden to see image'). Note that this is true for the owner of the channel or the channel's connections (who have the right permissions).

Also note that the 'forbidden to see image' can be seen in the photo app, and in the channel and network stream.

When viewing the photo's via the file browser app the photo (as a file) can be seen and the photo can be opened + downloaded.

**Comment by jeroenpraat** [comment]
I found out that it only affects hubzilla.nl. That's a standard installation on master. Hubuno.nl (a basic install (UNO) is working normally (also master). My development hub (on 1.12RC) is also working normal.

There are no failed database updates on hubzilla.nl. I checked it on multiple channels and they all had problems.

**Comment by jeroenpraat** [comment]
Forget it, it's a session issue.

**Comment by 20161220-rootfest** [comment]
(offtopic, but have to write here since [jeroenpraat.nl](https://jeroenpraat.nl/channel/jeroen/?f=&mid=3331cb758f7c909c2febb216338d27896065f0606da30db80e5235e697ac1e40@jeroenpraat.nl) is read-only).
It's a pity that you closed hubuno.nl with such short notification period. Hubzilla is not just a simple social network, the ability to create webpages and wikis on channel is its distinguishing feature.

**Comment by zotlabs** [comment]
All you have to do is go to another hubzilla site and import your existing channel and all its content before the old site shuts down completely. This is Hubzilla's distinguishing feature.

**Comment by jeroenpraat** [comment]
It is just that Hubuno had NOT those webpages and wikis features, cause it was a Basic Hubzilla installation. It was for people looking for casual/simple social networking. But I changed my mind about Hubzilla suitability for simple social networking. There are better free options for that (don't let the freeweb duplicate things, but complement eachother).

As Mike pointed out, there are enough other Hubzilla hubs. I also started a Mastodon instance (partly as a replacement for people looking for simple social network experience and also cause I prefer for myself a more Twitter like experience). https://todon.nl (it has mostly the same TOS as Hubuno)

Finally I like to say that 1 month notice is not that short for a hub that only had a few active users (+ it is my server). And also you had the opportunity to communicate using other channels, so without hijacking a Github issue.
